kolibri/core/content/api.py

```diff
@@ -610,7 +610,21 @@ def get_resume_queryset(request, queryset):
         .exclude(progress=1)
         .values_list("content_id", flat=True)
     )
-    return queryset.filter(content_id__in=content_ids)
+    queryset = queryset.filter(content_id__in=content_ids)
+    queryset = queryset.annotate(
+        last_interacted=Subquery(
+            ContentSummaryLog.objects.filter(
+                user=request.user, progress__gt=0, content_id=OuterRef("content_id")
+            )
+            .exclude(progress=1)
+            .values_list("end_timestamp")[:1]
+        )
+    )
+    queryset = queryset.order_by("-last_interacted")
```

Comment:
We don't need to do the `order_by` here any more because we're using the `OrderingFilter` below to do it.